pkgdb/src/main.cc

```diff
@@ -26,6 +26,11 @@
 #include "flox/eval.hh"
 #include "flox/parse/command.hh"
 #include "flox/pkgdb/command.hh"
+
+#ifndef __APPLE__
+#  include "flox/pkgdb/metrics.hh"
+#endif
```

Comment:
nit(nb): is there a __LINUX__ equivalent? The makefiles use `if linux` c files using `if not apple` is a bit confusing.

Reply:
nit: could we defer the conditional check to the metrics class? i.e. implement its init as a noop on mac, that might also make some of the makefile changes unnecessary.

Reply:
Ok, I think that's a tad cleaner?  at least it gets rid of the Makefile `filter-out` stuff.  And yes, switched to a `__linux__` macro.

Reply:
yes, i think this is cleaner now.